content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -46,5 +56,7 @@ jenkins:
 
 === In official Docker images
 
-Official link:https://github.com/jenkinsci/docker[Jenkins Docker image] includes plugin management scripts which allow preinstalling plugins from the Experimental update center.
+The official link:https://github.com/jenkinsci/docker[Jenkins Docker image] includes plugin management scripts which allow preinstalling plugins from the Experimental update center. 
+The container images support managing plugin versions as code, so that the administrator can define a container image that includes the exact plugin they require.
```

Comment:
```suggestion
The official link:https://github.com/jenkinsci/docker[Jenkins Docker image] includes plugin management scripts, which allow preinstalling plugins from the _experimental update center_. 
The container images support managing plugin versions as code, so that the administrator can define a container image including the exact plugin they require.
```
suggestion for adding italicized formatting, and updating some text